tests/run.bats

```diff
@@ -838,7 +838,7 @@ $output"
 	skip_if_no_runtime
 
 	_prefetch alpine
-	run_buildah from --quiet --pull=false $WITH_POLICY_JSON alpine
+	run_buildah from --quiet --pull=ifmissing $WITH_POLICY_JSON alpine
```

Comment:
This one shouldn't need changing - the named image is prefetched.  Does it start to fail without this change?